scripts/session.js

```diff
@@ -140,16 +136,17 @@ ws.onmessage = function(evt){
     Cookies.remove('point');
   }
 
+  // show all votes
   if(my_received_message.type == "show_all_votes"){
     $("td[id]").each(function(){
       this.className = '';
     });
   }
 };
 
-// add event handler for diconnection
+// reconnect when server is disconnected
 ws.onclose= function(evt){
-  alert('Server Diconnected');
+  window.location.reload();
```

Comment:
We may look for a better way to handle auto reconnect. See also: https://github.com/joewalnes/reconnecting-websocket